ts/Dashboards/Board.ts

```diff
@@ -662,15 +662,15 @@ namespace Board {
          *
          * Try it:
          *
-         * {@link https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/dashboards/demo/component-highcharts/  | Highcharts component}
+         * {@link https://www.highcharts.com/samples/embed/dashboards/components/component-highcharts | Highcharts component}
```

Comment:
What with the other `.ts` files in the `Dashboards` direcotry? Shouldn't we also change them?